static/js/models/database.js

```diff
@@ -9,23 +9,25 @@ var dbConfig = {
     user: 'client',
     password: passwords.getMySQLClientPassword(),
     database: 'polyscore', 
-    multipleStatements: true
+    multipleStatements: true,
+    packetSize: 16384 ,
+    connectionLimit: 10
 }
 
-var connection;
+var pool;
 
 function handleDisconnect() {
-    connection = mysql.createConnection(dbConfig);
+    pool = mysql.createPool(dbConfig);
 
-    console.log("We ran handleDisconnect...")
-    connection.connect(function(err) {
-        if (err) {
-            console.log('error when connecting to db: ', err);
-            setTimeout(handleDisconnect, 2000);
-        }
-    });
+    // console.log("We ran handleDisconnect...")
```

Comment:
Can this commented code be removed?

Reply:
Not yet, I'm still kind of testing this pool code and I want to keep this here just in case we have to go back to it